stock_cycle_count/models/stock_cycle_count.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class StockCycleCount(models.Model):
+    _name = 'stock.cycle.count'
+    _inherit = 'mail.thread'
+
+    @api.one
+    def _count_inventory_adj(self):
+        self.inventory_adj_count = len(self.stock_adjustment_ids)
+
+    @api.model
+    def create(self, vals):
+        vals['name'] = self.env['ir.sequence'].next_by_code(
+            'stock.cycle.count') or ''
+        return super(StockCycleCount, self).create(vals)
+
+    @api.model
+    def _company_get(self):
+        company_id = self.env['res.company']._company_default_get(self._name)
+        return company_id
+
+    name = fields.Char(string='Name', readonly=True)
```

Comment:
What do you mean? the label? I just like to use almost always labels to make the code more readable.